# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/transform-3d/reftest.list
# Commit: b7f88e5c537b
# Full Hash: b7f88e5c537bb7d64ba9d4cb58b052650d510638
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-08 17:32:27
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers,kvark
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r 8e9c1ffdeed8 -r b7f88e5c537b layout/reftests/transform-3d/reftest.list
--- a/layout/reftests/transform-3d/reftest.list	Mon Feb 07 22:02:35 2022 +0000
+++ b/layout/reftests/transform-3d/reftest.list	Mon Feb 07 22:14:47 2022 +0000
@@ -82,9 +82,9 @@
 fuzzy(0-1,0-10000) == opacity-preserve3d-4.html opacity-preserve3d-4-ref.html
 == opacity-preserve3d-5.html opacity-preserve3d-5-ref.html
 == snap-perspective-1.html snap-perspective-1-ref.html
-fuzzy-if(Android,0-8,0-1) == mask-layer-1.html mask-layer-ref.html
-fuzzy-if(Android,0-8,0-1) == mask-layer-2.html mask-layer-ref.html
-fuzzy(0-16,0-132) == mask-layer-3.html mask-layer-ref.html
+fuzzy(0-120,0-590) == mask-layer-1.html mask-layer-ref.html
+fuzzy(0-120,0-590) == mask-layer-2.html mask-layer-ref.html
+fuzzy(0-120,0-590) == mask-layer-3.html mask-layer-ref.html
 == split-intersect1.html split-intersect1-ref.html
 fuzzy(0-255,0-150) fails-if(useDrawSnapshot) == split-intersect2.html split-intersect2-ref.html
 fuzzy(0-255,0-100) fails-if(useDrawSnapshot) == split-non-ortho1.html split-non-ortho1-ref.html
@@ -94,7 +94,7 @@
 == intermediate-1.html intermediate-1-ref.html
 == preserves3d-nested-filter-1.html preserves3d-nested-filter-1-ref.html
 != preserve3d-scale.html about:blank
-fuzzy(0-51,0-1154) == preserve3d-scale.html preserve3d-scale-ref.html
+fuzzy(0-50,0-1460) == preserve3d-scale.html preserve3d-scale-ref.html
 fuzzy(0-1,0-5) == perspective-overflow-1.html perspective-overflow-1-ref.html
 == perspective-overflow-2.html perspective-overflow-2-ref.html
 == 1544995-1.html 1544995-1-ref.html